airflow/providers/postgres/hooks/postgres.py

```diff
@@ -159,6 +172,10 @@ def get_conn(self) -> connection:
             ]:
                 conn_args[arg_name] = arg_val
 
+        client_encoding = conn.extra_dejson.get("client_encoding")
+        if isinstance(client_encoding, str):
+            conn_args["client_encoding"] = client_encoding
+
```

Comment:
Is this related to description of the PR?

Reply:
I had to change this to make `test_get_uri` test pass